src/features/RegisterPage/index.ts

```diff
@@ -0,0 +1,3 @@
+import RegisterPage from './RegisterPage';
+
+export default RegisterPage;
```

Comment:
This seems like an unnecessary file. Now there are two files exporting RegisterPage. Exporting it in and importing from the RegisterPage.tsx should be sufficient.
```suggestion
```

Reply:
Could everything be in the index.tsx then?